packages/shorebird_cli/lib/src/shorebird_environment.dart

```diff
@@ -63,4 +63,16 @@ abstract class ShorebirdEnvironment {
           'flutter',
         ),
       );
+
+  static File get genSnapshotFile => File(
```

Comment:
This will end up with the wrong one when we specify --local-engine-src-path (which is the only way which can work today).  flutter_tools has a way by which they get all their paths like this from some sort of shared config object which knows how to respect --local-engine.  We don't necessarily need something fancy, but this will produce the wrong result for now.   I guess that's also fine as I can hack it locally for the demo, but I would at least leave a fixme to say this doesn't support --local-engine.

Reply:
Added a TODO, will fix in an upcoming PR